config_handler.py

```diff
@@ -5,7 +5,7 @@
 __blacklist_base_config = ["is_managed", "is_demo", "services_to_notify"]
 __config = None
 
-BASE_CONFIG_PATH = "base_config.json"
+BASE_CONFIG_PATH = "/base-config/base_config.json"
 CURRENT_CONFIG_PATH = "current_config.json"
```

Comment:
i think these should be the other way around. the base config is the template and used to create the current_config if i remember correct. The current should be the one with the current values

Reply:
absolutely! did it